samples/generate-from-code-sample/README.md

```diff
@@ -0,0 +1,42 @@
+## Prerequisites
+You will need:
+1. [JDK 1.8][jdk]
+2. [Maven 3.x][maven]
+
```

Comment:
recommend to add the description of configuration

Reply:
ok, I will the description of configuration in the next commit.